codemods/run-codemod.ts

```diff
@@ -0,0 +1,191 @@
+import { execSync } from "child_process";
+import * as fs from "fs";
+import * as path from "path";
+import { checkbox, confirm, input, select } from "@inquirer/prompts";
+
+const codemodsDir: string = path.join(process.cwd(), "codemods");
+
+const codemodDescriptions: { [key: string]: string } = {
+    "deprecate-v2-tokens": "This is to deprecate previous tokens to V2",
+    "migrate-colour":
+        "This is to migrate V2_Color to its respective Colour tokens are usages.",
+    "migrate-layout":
+        "This is to migrate V2_Layout to its new Layout component and respective column usages",
+    "migrate-mediaquery":
+        "This is to migrate V2 mediaquery helper function to V3 and its new specs.",
+    "migrate-text":
+        "This is to migrate V2_text to its new Typography Components and previous usages.",
+};
+
+// Shortcuts
+const predefinedPaths: { [key: string]: string } = {
+    Src: "src",
+    Codebase: ".",
+    TestData: "test-data/test-data.tsx",
+};
+
+// Return codemods in codemod folder
+function listCodemods(): { name: string; value: string }[] {
+    const codemods: string[] = fs.readdirSync(codemodsDir).filter((folder) => {
+        return fs.lstatSync(path.join(codemodsDir, folder)).isDirectory();
+    });
+
+    const options = codemods.map((mod) => ({
+        name: codemodDescriptions[mod]
+            ? `${mod} - ${codemodDescriptions[mod]}`
+            : `${mod}`,
+        value: mod,
+    }));
+
+    options.push({ name: "Exit", value: "exit" });
+    return options;
+}
+
+function runCodemods(
+    codemodsToRun: string[],
+    outputPath: string,
+    mapping?: string
```

Comment:
this argument can be generic to support other codemods that also have options in the future

```suggestion
    options?: { "migrate-colour"?: Theme }
```